### PR TITLE
Search employees

### PR DESCRIPTION
**Pull request by polyakovigor**
Add search feature
No tests in the current moment

### CODE REVIEW

File: app/controllers/employees_controller.rb [hunk 1]
```diff
@@ -4,14 +4,13 @@ class EmployeesController < ApplicationController
   # TODO
   def index
     @skills = Skill.all
-    @employees = Employee.all
+    search_employees
```
Comment (Tivgres) [inline app/controllers/employees_controller.rb]: better to use callback!?

File: app/controllers/employees_controller.rb [hunk 2]
```diff
@@ -4,14 +4,13 @@ class EmployeesController < ApplicationController
...
   end
 
   # TODO
   def show
     @skills = Skill.all
-    @employees = Employee.all
-    @employee = Employee.find(params[:id])
+    search_employees
```
Comment (Tivgres) [inline app/controllers/employees_controller.rb]: better to use callback!?

File: app/controllers/employees_controller.rb (hunk 1 above)
Comment (polyakovigor) [reply app/controllers/employees_controller.rb]: it is a temporary solution.

File: app/controllers/employees_controller.rb (hunk 2 above)
Comment (polyakovigor) [reply app/controllers/employees_controller.rb]: it is a temporary solution.

File: app/controllers/employees_controller.rb (hunk 1 above)
Comment (Tivgres) [reply app/controllers/employees_controller.rb]: Yeah, but I think better to use callback at head of class, nope?

File: app/controllers/employees_controller.rb (hunk 2 above)
Comment (Tivgres) [reply app/controllers/employees_controller.rb]: Yeah, but I think better to use callback at head of class, nope?

File: app/views/layouts/application.html.haml [hunk 3]
```diff
@@ -10,10 +10,11 @@
     %header
       %nav.navbar.navbar-light.bg-light
         .navbar-brand= link_to 'FaceIT', root_path
-        %form.form-inline
+        = form_tag employees_path, method: :get, class: 'form-inline' do
```
Comment (drkmen) [inline app/views/layouts/application.html.haml]: you can use `= form_tag ''`, to mentione the form send query to current page

File: app/controllers/employees_controller.rb [hunk 4]
```diff
@@ -56,6 +56,7 @@ def load_data
     @employees = Employee.filter(params.reject { |_, v| v.blank? }.slice(:role, :office, :department))
     @employees = Employee.filter_skills(@employees, params[:skills]) if params[:skills]
     @employees = @employees.to_a.group_by(&:department) unless @employees.empty?
+    @employees = Employee.search(params[:term]).to_a.group_by(&:department) if params[:term]
```
Comment (drkmen) [inline app/controllers/employees_controller.rb]: move it to `#load_data`

Comment (polyakovigor) [reply app/controllers/employees_controller.rb]: already there

File: app/views/layouts/application.html.haml (hunk 3 above)
Comment (polyakovigor) [reply app/views/layouts/application.html.haml]: done

**Pull request merged**